### ISSUE TITLE
ALTER COLUMN "default_value" SET DEFAULT 'b'0''

### ISSUE THREAD

**Issue by ScoobyMax** [AnatolyUss/nmig#89]
A lot of errors on my migration:

```

        --[DefaultProcessor::default] error: ошибка синтаксиса (примерное положение: "1")

        SQL: ALTER TABLE "public"."project_status" ALTER COLUMN "can_delete" SET DEFAULT 'b'1'';


        --[DefaultProcessor::default] error: ошибка синтаксиса (примерное положение: "0")

        SQL: ALTER TABLE "public"."project_status" ALTER COLUMN "show_in_status_history" SET DEFAULT 'b'0'';

```

**Comment by ScoobyMax** [comment]
Source mysql table columns like:
```
    deleted                   bit default b'0'              not null,
    completed              bit default b'0'              not null,
    readonly                 bit default b'0'              not null,
    warning                  bit default b'0'              not null,
```

I guess that the boolean type should turn out

**Comment by AnatolyUss** [comment]
Hello Maxim,
I'll check it asap. (today-tomorrow)

**Comment by AnatolyUss** [comment]
Hello again,
The issue is fixed in the latest release (<code>Nmig v5.6.0</code>)
Please, pull latest changes from <code>master</code>, If still relevant.
Sorry for the delay.
